iocore/net/SSLNetVConnection.cc

```diff
@@ -1238,12 +1238,12 @@ SSLNetVConnection::sslServerHandShakeEvent(int &err)
     if (SSL_get_all_async_fds(ssl, &waitfd, &numfds) && numfds > 0) {
       // Temporarily disable regular net
       read_disable(nh, this);
+      write_disable(nh, this);
       this->ep.stop(); // Modify used in read_disable doesn't work for edge triggered epol
       // Have to have the read NetState enabled because we are using it for the signal vc
-      read.enabled = true;
-      write_disable(nh, this);
+      read.enabled       = true;
       PollDescriptor *pd = get_PollDescriptor(this_ethread());
-      this->ep.start(pd, waitfd, this, EVENTIO_READ);
+      this->ep.start(pd, waitfd, static_cast<NetEvent *>(this), EVENTIO_READ);
       this->ep.type = EVENTIO_READWRITE_VC;
```

Comment:
IMO, replace these lines with `read.triggered = false` and `write.triggered = false` is more suitable here. Also the L1252 to L1258 can be deleted.